src/transformers/modeling_utils.py

```diff
@@ -932,6 +933,18 @@ def prune_heads(self, heads_to_prune: Dict[int, List[int]]):
 
         self.base_model._prune_heads(heads_to_prune)
 
+    def gradient_checkpointing_enable(self, flag: bool = True):
+        """
+        Activates or deactivates gradient checkpointing for the current model.
```

Comment:
```suggestion
        Activates or deactivates gradient checkpointing for the current model.
        
        Note that in other frameworks this feature can be referred to as "activation checkpointing" or "checkpoint activations".
```

Let's make it easier for users who are familiar with this feature by another name to find it in `transformers`.